### PR TITLE
Make rounded corners easier to see

### PR DESCRIPTION
**Pull request by littlewhitecloud**
I love this repo!!! And maybe I will use it in my project. 
There is a bit change to make rounded corners easier to see.

### CODE REVIEW

File: dialogs.py [hunk 1]
```diff
@@ -102,80 +107,82 @@ def on_button(value):
...
 if __name__ == "__main__":
     window = tk.Tk()
-
-    window.tk.call("source", "sun-valley.tcl")
-    window.tk.call("set_theme", "dark")
-
     window.geometry("600x600")
 
-    show_message("No WiFi connection", "Check your connection and try again.")
+    ask_allow_block("No WiFi connection", "Check your connection and try again.", theme = "dark")
```
Comment (rdbende) [inline dialogs.py]:
```suggestion
show_message("No WiFi connection", "Check your connection and try again.")
```

This is totally unrelated.

File: dialogs.py [hunk 2]
```diff
@@ -3,8 +3,13 @@
 from functools import partial
 
 
-def popup(parent, title, details, icon, *, buttons):
+def popup(parent, title, details, icon, *, buttons, theme):
     dialog = tk.Toplevel()
+    window.tk.call("source", "sun-valley.tcl")
+    
+    window.tk.call("set_theme", "%s" % theme)
```
Comment (rdbende) [inline dialogs.py]:
```suggestion
```
This breaks when showing a popup more than once. The theme can only be loaded once.
Please also remove the `theme` option.

File: dialogs.py (hunk 1 above)
Comment (littlewhitecloud) [reply dialogs.py]: Sorry, I was testing the messagebox and I forgot to change it back.

File: dialogs.py [hunk 3]
```diff
@@ -3,8 +3,13 @@
...
+    if theme == "light": dialog.attributes("-transparentcolor", "#fafafa")
+    else: dialog.attributes("-transparentcolor", "#1c1c1c")
```
Comment (rdbende) [inline dialogs.py]:
```suggestion
    if parent.tk.call("tk", "windowingsystem") == "win32":
        transparent_color = parent.tk.call("ttk::style", "lookup", "TFrame", "-background")
        dialog.wm_attributes("-transparentcolor", transparent_color)
```
I don't know whether this works at all, but should definitely be Windows-only

**Comment by littlewhitecloud** [comment]
image
```python
AttributeError: 'NoneType' object has no attribute 'tk'
```

File: dialogs.py (hunk 3 above)
Comment (rdbende) [reply dialogs.py]: Yes, the `-transparentcolor` option is Windows-only

**Comment by rdbende** [comment]
Oh, yeah, because the parent can be `None`

**Comment by rdbende** [comment]
Nice. Now it automatically picks up the theme and makes the corners transparent based on the theme background color. Could you show a screenshot of this on Windows?

**Comment by littlewhitecloud** [comment]
Wait a minute.

**Comment by littlewhitecloud** [comment]
image
image

**Comment by rdbende** [comment]
Uhh, what's this weird font?

The four black dots at the corners of the button are because the button background is the same color as the transparent color of the dialog. I will fix it in the theme.

**Comment by littlewhitecloud** [comment]
> Uhh, what's this weird font?
> 
> The four black dots at the corners of the button are because the button background is the same color as the transparent color of the dialog. I will fix it in the theme.

Maybe it is because of I am using windows 10 and it doesn’t have a correct seoge font.

**Comment by littlewhitecloud** [comment]
Moved to #5
